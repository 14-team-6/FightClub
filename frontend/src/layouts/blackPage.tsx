import React, { ReactNode, FC } from 'react';
import styled from 'styled-components';
import { MAIN_BLACK } from '../../consts/styles';

const Wrap = styled.div`
  background-color: ${MAIN_BLACK};
  height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
`;

type Props = {
  children?: ReactNode
};

export const BlackPageLayout: FC<Props> = ({ children }) => {
  return <Wrap>
    { children }
  </Wrap>;
};
