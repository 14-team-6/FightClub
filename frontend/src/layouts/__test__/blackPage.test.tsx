import React from 'react';
import renderer from 'react-test-renderer';
import { BlackPageLayout } from '../blackPage';

describe('black page layout should render', () => {
  it('should render', () => {
    const tree = renderer
      .create(<BlackPageLayout/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
