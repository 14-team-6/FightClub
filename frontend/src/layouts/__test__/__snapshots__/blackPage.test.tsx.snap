// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`black page layout should render should render 1`] = `
<div
  className="sc-bczRLJ kkcMAq"
/>
`;
